Commands/balance.go

```diff
@@ -0,0 +1,12 @@
+package Commands
```

Comment:
Package names should be lowercase and, typically, singular:

https://blog.golang.org/package-names

Also, a good rule of thumb is to think about the package name as part of the types and values the package exposes, since most of the time the package name is used with those things. So, for example, instead of `command.CommandRegistry` you might want to use `command.Registry`, since the package name communicates context about what kind of registry it is.

Reply:
Refactored. Thanks!